config.go

```diff
@@ -422,9 +464,45 @@ var configValidationRules = []configValidator{
 	},
 
 	func(settings *Settings) error {
-		if settings.LogBuffering.DurationSeconds == 0 {
-			settings.LogBuffering.DurationSeconds = defaultLogBufferingDuration
+		if settings.LoggerConfig.Buffer.DurationSeconds == 0 {
+			settings.LoggerConfig.Buffer.DurationSeconds = defaultLogBufferingDuration
 		}
+
+		if settings.LoggerConfig.DefaultLevel == "" {
+			settings.LoggerConfig.DefaultLevel = "info"
+		}
+
+		if settings.LoggerConfig.ThresholdLevel == "" {
+			settings.LoggerConfig.ThresholdLevel = "debug"
+		}
+
+		info := settings.LoggerConfig.Info()
+		if !info.Valid() {
+			return errors.Errorf("logging level configuration is not valid [%+v]", info)
+		}
+
+		return nil
+	},
+
+	func(settings *Settings) error {
+		if settings.Slack.Token != "" {
+			if settings.Slack.Options.Channel == "" {
+				settings.Slack.Options.Channel = "#mci-alerts"
```

Comment:
#evergreen-alerts, pretty please

Reply:
I want something that's clear that it's not something that users would be interested in.